### ISSUE TITLE
increased non-HDL cholesterol concentration 

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#6092]
**Preferred term label:**

increased non-HDL cholesterol concentration 

**Synonyms**

elevated non-HDL cholesterol concentration 

**Definition (free text, please give PubMed ID)**

Increase above normal levels of non-HDL cholesterol in the blood.  Non-HDL cholesterol is total cholesterol minus high-density lipoprotein HDL-cholesterol (high-density lipoprotein-cholesterol).  PMID:1596847

**Parent term (use hpo.jax.org/app)**

Abnormal circulating cholesterol concentration (HP:0003107)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

RGD:gthayman

**Comment by pnrobinson** [comment]
sorry for the delay, added this term as suggested.
